jaxrs/src/main/java/io/airlift/jaxrs/tracing/TracingServletFilter.java

```diff
@@ -1,44 +1,152 @@
 package io.airlift.jaxrs.tracing;
```

Comment:
This is the wrong package/module. It should go in the HTTP server module.

Reply:
sure, I can move it

Reply:
Moved.